.github/workflows/command-prdoc.yml

```diff
@@ -0,0 +1,78 @@
+name: Command PrDoc
+
+on:
+  workflow_dispatch:
+    inputs:
+      pr:
+        type: number
+        description: Number of the Pull Request
+        required: true
+      bump:
+        type: choice
+        description: Default bump level for all crates
+        default: "TODO"
+        required: true
+        options:
+          - "TODO"
+          - "no change"
+          - "patch"
+          - "minor"
+          - "major"
+      audience:
+        type: choice
+        description: Audience of the PrDoc
+        default: "TODO"
+        required: true
+        options:
+          - "TODO"
+          - "Runtime Dev"
+          - "Runtime User"
+          - "Node Dev"
+          - "Node User"
+      overwrite:
+        type: choice
+        description: Overwrite existing PrDoc
+        default: "true"
+        required: true
+        options:
+          - "true"
+          - "false"
+
+concurrency:
+  group: command-prdoc
+  cancel-in-progress: true
+
+jobs:
+  cmd-prdoc:
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - name: Download repo
+        uses: actions/checkout@v4
+      - name: Install gh cli
+        id: gh
+        uses: ./.github/actions/set-up-gh
+        with:
+          pr-number: ${{ inputs.pr }}
+          GH_TOKEN: ${{ github.token }}
+      - name: Generate PrDoc
+        run: |
+          python3 -m pip install -q cargo-workspace PyGithub whatthepatch pyyaml toml
+
+          python3 .github/scripts/generate-prdoc.py --pr "${{ inputs.pr }}" --bump "${{ inputs.bump }}" --audience "${{ inputs.audience }}" --force "${{ inputs.overwrite }}"
```

Comment:
may be you could also alternatively add an option to start it with command?
like here https://github.com/polkadot-fellows/runtimes/pull/406/files#diff-38149785bc1a430631c477eabc9b0970804d2c9433504a67574d0b9d38461ba6R120

```yaml
  issue_comment: # listen for comments on issues
    types: [created]
```

Reply:
I have a `type` set on all inputs to either `number` or `choice` so i dont think that there can be any query injection.

But i would also prefer comments, as outsiders can probably not run CI actions. Not sure why the comment approach was deprecated?

Reply:
it wasn't deprecated, we just tried to move to GHA faster/cheaper, and that was one of the fast and secure ways